src/sentry/static/sentry/app/views/settings/incidentRules/types.tsx

```diff
@@ -163,3 +165,35 @@ export type UnsavedAction = {
    */
   integrationId?: number | null;
 };
+
+type OptionConfig = {
+  aggregations: AggregationKey[];
+  fields: LooseFieldKey[];
+};
+
+/**
+ * Allowed error aggregations for alerts
+ */
+export const errorFieldConfig: OptionConfig = {
```

Comment:
These probably should live somewhere else outside of the `types` file

Reply:
there was a constants file right next to it and i'm just blind